app/validators/notation.js

```diff
@@ -0,0 +1,18 @@
+import _ from "npm:lodash"
+
+export default function validateNotation() {
+  return (key, oldValue) => {
+    let checkStandardNotation = _.map(Object.values(oldValue), standard => {
```

Comment:
since this if just returns a boolean, you can shorten it to something like `standard => _.isPresent(standard.statementNotation)`

Reply:
no such method at _.isPresent...went through all array methods and object methods.  Thought about using isUndefined, but it comes back as an empty string when there is no entry.

Reply:
if it's returning an empty string, my guess is something else is going on. _.isUndefined should return a boolean. I might try it again and see if it was something else.

Reply:
Think there was a misunderstanding because I didn't communicate effectively.  isUndefined does not return an empty string.  If standard.statementNotation is empty it comes back as an empty string which is NOT undefined so even if there is no notation

Reply:
refactored to this:

```  return (key, oldValue) => {
    let checkStandardNotation = _.filter(Object.values(oldValue), standard => {
      return standard.statementNotation
    })

    let isValid = checkStandardNotation.length/Object.values(oldValue).length

    return isValid >= 0.3 ? true : "notation needed"
  }```

Reply:
That is definitely better! I'd change the return standard.statementNotation to something that looks like a check. E.g. `return !_.isBlank(standard.statementNotation)` Just returning it in a filter is a truthy type of comparison and I never know exactly waht's going to qualify. Making it explicit makes it easier to reasona bout without knowing hte in and outs of what filter considers truthy.

Reply:
Did it!